boards/raytac/an7002q_db/doc/index.rst

```diff
@@ -0,0 +1,267 @@
+* COEX_STATUS1 = **P0.29**
+* COEX_GRANT = **P0.24**
+
+Security components
+-------------------
+
+The following security components are available:
+
+* Implementation Defined Attribution Unit (`IDAU`_) on the application core.
+
+  The IDAU is implemented with the System Protection Unit and is used to define
+  secure and non-secure memory maps. By default, the entire memory space
+  (Flash, SRAM, and peripheral address space) is defined to be secure-accessible only.
+
+* Secure boot.
+
+Programming and Debugging
+*************************
+
+.. zephyr:board-supported-runners::
+
+The nRF5340 application core supports the Armv8-M Security Extension.
+Applications built for the ``raytac_an7002q_db/nrf5340/cpuapp`` board boot by default in the
+secure state.
+
+The nRF5340 network core does not support the Armv8-M Security Extension.
+nRF5340 IDAU can configure bus accesses by the nRF5340 network core to have the secure
+attribute set. This allows to build and run secure-only applications on the nRF5340 SoC.
+
+Building Secure/Non-Secure Zephyr applications with Arm |reg| TrustZone |reg|
+=============================================================================
+
+Applications on the nRF5340 may contain a Secure and a Non-Secure firmware
+image for the application core. The Secure image can be built using either
+Zephyr or `Trusted Firmware M`_ (TF-M). Non-Secure firmware
+images are always built using Zephyr. The two alternatives are described below.
+
+.. note::
+
+   By default, SPE for the nRF5340 application core is built using TF-M.
+
+Building the Secure firmware with TF-M
+--------------------------------------
+
+The process to build the Secure firmware image using TF-M and the Non-Secure
+firmware image using Zephyr requires the following steps:
+
+1. Build the Non-Secure Zephyr application
+   for the application core using ``-DBOARD=raytac_an7002q_db/nrf5340/cpuapp/ns``.
+   To invoke the building of TF-M the Zephyr build system requires the
+   Kconfig option ``BUILD_WITH_TFM`` to be enabled, which is done by
+   default when building Zephyr as a Non-Secure application.
+   The Zephyr build system will perform the following steps automatically:
+
+      * Build the Non-Secure firmware image as a regular Zephyr application
+      * Build a TF-M (secure) firmware image
+      * Merge the output image binaries together
+      * Optionally build a bootloader image (MCUboot)
+
+.. note::
+
+   Depending on the TF-M configuration, an application DTS overlay may be
+   required, to adjust the Non-Secure image Flash and SRAM starting address
+   and sizes.
+
+2. Build the application firmware for the network core using
+   ``-DBOARD=raytac_an7002q_db/nrf5340/cpunet``.
+
+Building the Secure firmware using Zephyr
+-----------------------------------------
+
+The process to build the Secure and the Non-Secure firmware images
+using Zephyr requires the following steps:
+
+1. Build the Secure Zephyr application for the application core
```

Comment:
@Vge0rge @tomi-font should probably rewrite this for nrf5340dk to use the sphinx goal build tag things then update other vendor boards